src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,72 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "movie_sessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    private Movie movie;
+    @ManyToOne
+    private CinemaHall cinemaHall;
+    private LocalDateTime showTime;
+
+    public MovieSession() {
+    }
+
+    public MovieSession(Movie movie, CinemaHall cinemaHall) {
+        this.movie = movie;
+        this.cinemaHall = cinemaHall;
+    }
```

Comment:
The constructor `MovieSession(Movie movie, CinemaHall cinemaHall)` does not initialize the `showTime` field, which is a crucial part of a `MovieSession`. You should consider adding a parameter to set the `showTime` upon creating an instance.

Reply:
+